### DISCUSSION TITLE
Custom button

### DISCUSSION THREAD

**Discussion by rkosick** [outdoorbits/little-backup-box#185] [Ideas]
Hello,
there was an option to create custom backup buttons in an early version. I would like to see a custom button again, just to perform additional commands.
Background: I would like to have a button to start a wireguard connection to my network at home. So I don't have to connect over ssh seperatly to do it.

**Comment by outdoorbits** [comment]
The custom buttons just pointed to empty script files, own coding was 
needed. This was incompatible to a configurable and update-able 
solution. Do you have to code you would need to execute for your 
connection?Message ID: 
***@***.***>

**Reply by rkosick** [reply]
I understand. I don't need to code anything. Just want to execute a command like "wg-quick up connection_xy"

**Comment by outdoorbits** [comment]
But I expect you would need a config-file and a certificate?

**Reply by rkosick** [reply]
I don't understand your question. I just need to execute a command like I would be after connection via ssh. It was just an idea. I also can modify the code where I need it. I just was like thinking about a custom button that executes a custom command. No worries. Thanks for your great work.

**Comment by outdoorbits** [comment]
Yes, I just try to understand. You would need an option to insert code, 
even if it's one line only. But don't you need additional config-files 
to establish that connection?

**Reply by rkosick** [reply]
Thats right. Don't know if it would be interesting for other people or just me.

**Comment by outdoorbits** [comment]
I do not have any experience with wireguard. But maybe you could implement this very easy by 
`sudo crontab -e` 
to open the cron-file of root. (Please do not use this of user pi, because content will be removed by lbb in case of changes or an update.)
Now you can write a line
`@reboot wg-quick up connection_xy`
It will be executed after reboot.
But if you could give me detailed information (packages needed, config-files...) maybe I could include this...

**Reply by rkosick** [reply]
Yes I thought about something similar. That would activate it every time. I think I'll be fine with that.
Thanks.
